### DISCUSSION TITLE
Connection Not Secured (SSL not being applied on my website)

### DISCUSSION THREAD

**Discussion by hallectj** [vercel/community#5052] [Help]
### Summary

So like others, I have a custom domain. I successfully changed the name servers, I got the "Good news! Your DNS records are set up correctly" message. I look at my SSL and it is there as in I an see it on my Vercel dashboard, I see the : CAA  0 issue "letsencrypt.org",  What else do I need to do?  Do I just delete my domain and re-add it?  Can someone suggest things I need to do so?  I know this isn't much to go off of, but I'm lost as to what to do.

![sslNotApplid](https://github.com/vercel/community/assets/15127702/d05e2b4f-ad4d-442e-a908-a5436318c0c8)

![ssl_certs_dashboard](https://github.com/vercel/community/assets/15127702/3cd367c6-632b-43b6-9f90-e164ecbb6f3c)
![success_dns](https://github.com/vercel/community/assets/15127702/d7cf16c2-daa4-4ee4-af1e-522995aaba74)

### Steps to Reproduce

There is no "error" message, my site doesn't have the SSL applied, even though it is on my Vercel dashboard.

**Comment by lyndonb1** [comment]
How did you figure this out, love backthennow.com by the way?
